README.md

```diff
@@ -36,9 +36,9 @@ let client =
   ReasonApollo.createApolloClient(~link=httpLink, ~cache=inMemoryCache, ());
 
 let app =
- <ReasonApolloHooks.ApolloProvider client>
+ <ApolloHooks.ApolloProvider client>
```

Comment:
I guess we can change this to `ApolloHooks.Provider`? What do you think?

Reply:
Agree